indico/modules/events/abstracts/views.py

```diff
@@ -78,6 +78,14 @@ class WPDisplayAbstracts(WPDisplayAbstractsBase):
     menu_entry_name = 'call_for_abstracts'
 
 
+class WPDisplayCallForAbstracts(WPDisplayAbstracts):
+    template_prefix = WPDisplayAbstracts.template_prefix + 'display/'
+
+    def getJSFiles(self):
+        return (WPDisplayAbstractsBase.getJSFiles(self) +
+                self._asset_env['modules_event_display_js'].urls())
```

Comment:
Fine for me, no strong opinion here. I think usually we just ended up wrapping it since once you have more than 2 elements it usually doesn't fit in one line and then it makes sense to have one item per line.